### PR TITLE
Add the possibility to change the ui basepath to avoid using nginx sub_filter

### PR DESCRIPTION
**Pull request by pletessier**
#460

### CODE REVIEW

File: docker/management-ui/files/entrypoint.sh [hunk 1]
```diff
@@ -20,6 +20,8 @@ setup() {
     echo "Configure management API and management UI"
     cat /var/www/html/constants.json.template | \
     sed "s#http://localhost:8093/#${MGMT_API_URL}#g;s#http://localhost:4200/#${MGMT_UI_URL}#g" > /var/www/html/constants.json
+
+    [[ -n $BASEPATH ]] && sed -i "s%<base href=\"/\">%<base href=\"$BASEPATH\">%g" /var/www/html/index.html
```
Comment (brasseld) [inline docker/management-ui/files/entrypoint.sh]: Where / How is defined BASEPATH ?

**Comment by brasseld** [comment]
Hi @pletessier 

Is this PR still relevant?

**Comment by pletessier** [comment]
Hi @brasseld 

I don't need it anymore since I'm using Traefik and "Host" rules rather than "Path".

But I still think it should be configurable, preferably in Angular.

**Comment by brasseld** [comment]
Unfortunately, the base path can not be configured using Angular so we are forced to rely on the RP in front of the UI to "rewrite" this path...

**Comment by tcompiegne** [comment]
A lot of configuration has been changed since this PR. 

To be closed.

**Pull request closed without merging**